src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  padding: 0;
```

Comment:
don't reset paddings for all elements. just use it for some special tags that you need to reset